<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="OscilloscopeWidget" name="oscilloscope" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>230</height>
       </size>
      </property>
      <property name="title">
       <string>GD3 Tag</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="gd3_tag_table">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>22</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>22</number>
         </attribute>
         <row>
          <property name="text">
           <string>Track Name:</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Game Name:</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>System Name:</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Track Author:</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Date:</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Converter Name:</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Notes:</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="sn76489_group">
      <property name="title">
       <string>SN76489 Control</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Stereo Strength:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sn76489_stereo_slider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sn76489_stereo_spin">
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_gmd_group">
      <property name="title">
       <string>GameBoy GMD Controll</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>RegisterNr.:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gb_reg_num">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>47</number>
         </property>
         <property name="displayIntegerBase">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gb_reg_value">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="displayIntegerBase">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="gb_write_reg">
         <property name="text">
          <string>Write Reg</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="log_output">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0,200);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="action_Export_Streaming_Data"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Export_Streaming_Data">
   <property name="text">
    <string>&amp;Export Streaming Data</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OscilloscopeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/oscilloscope_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
